app/routes/users/current.js

```diff
@@ -0,0 +1,25 @@
+import Route from '@ember/routing/route';
+import RSVP from 'rsvp';
+
+export default Route.extend({
+  model(params) {
+    return RSVP.hash({
+      user: this.get('store').findRecord('user', params.user_id),
+      allB: this.get('store').findAll('book')
```

Comment:
allB? does what? we talked about descriptive yet functional names. also remember this is unnecessary if you nested the routes correctly.